bench/src/bin/bulk.rs

```diff
@@ -79,7 +79,7 @@ async fn server(endpoint: quinn::Endpoint, opt: Opt) -> Result<()> {
                 };
                 trace!("stream established");
 
-                let _: tokio::task::JoinHandle<Result<()>> = tokio::spawn(async move {
+                tokio::spawn(async move {
```

Comment:
I couldn't think of any reason to bind this?

Reply:
I'd guess this was required at some point in the past to unify the return type. If it builds without it, great!